vortexasdk/endpoints/voyages_congestion_breakdown.py

```diff
@@ -83,6 +83,10 @@ def search(
         vessel_risk_level_excluded: Union[str, List[str]] = None,
         has_ship_to_ship: str = None,
         has_charterer: str = None,
+        intra_movements: str = None,
```

Comment:
These type annotations need to be updated.
A str can't be None by default

Reply:
I think the current implementation of the PythonSDK is expecting for all these optional fields to be by default `None` so the payload sent to the APIs is just what the client is explicit asking and the defaults are handle at API level

Reply:
That's fine, we can keep them as None. The point is that the type annotation is impossible (I am aware this is a problem all throughout the SDK but wanted to raise awareness so we all stop increasing the number of occurrences of this problem).
We can't type something as `str` then initialise it to `None`. It should be typed as `str | None`!